.travis.yml

```diff
@@ -3,25 +3,35 @@ jobs:
   include:
     - name: "Python 3.6.0 on Xenial Linux"
       python: 3.6
+
     - name: "Python 3.7.0 on Xenial Linux"
       python: 3.7
+
     - name: "Python 3.8.0 on Xenial Linux"
       python: 3.8     # this works for Linux but is ignored on macOS or Windows
+
     - name: "Python 3.7.4 on macOS"
       os: osx
       osx_image: xcode11.2  # Python 3.7.4 running on macOS 10.14.4
       language: shell       # 'language: python' is an error on Travis CI macOS
+      install:
+        - pip3 install --upgrade pip  # all three OSes agree about 'pip3'
+        - pip3 install black
+        - pip3 install --user ".[dev]" .
+
     - name: "Python 3.8.0 on Windows"
       os: windows           # Windows 10.0.17134 N/A Build 17134
       language: shell       # 'language: python' is an error on Travis CI Windows
       before_install:
         - choco install python --version 3.8.0
         - python -m pip install --upgrade pip
+        - pip3 install --no-deps torch===1.6.0 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
?

Reply:
Will remove this (I think we mixed something up in our merge strategy and thought this'll come after flair support (for which we need torch) is implemented